akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ClusterSharding.scala

```diff
@@ -258,7 +258,7 @@ class ClusterSharding(system: ExtendedActorSystem) extends Extension {
    * @param entityProps the `Props` of the entity actors that will be created by the `ShardRegion`
    * @param settings configuration settings, see [[ClusterShardingSettings]]
    * @param messageExtractor functions to extract the entity id, shard id, and the message to send to the
-   *   entity from the incoming message
+   *   entity from the incoming message, for `unhandled` messages it must return `null`
```

Comment:
This is also already mentioned on the `entityId` method of `MessageExtractor` which is what should return `null`, so this should refer to that method rather than the entire `messageExtractor `.